buildenv/jenkins/JenkinsfileBase

```diff
@@ -32,6 +32,9 @@ def setupEnv() {
 	ADOPTOPENJDK_REPO = params.ADOPTOPENJDK_REPO ? params.ADOPTOPENJDK_REPO : "https://github.com/AdoptOpenJDK/openjdk-tests.git"
 	OPENJ9_REPO = params.OPENJ9_REPO ? params.OPENJ9_REPO : "https://github.com/eclipse/openj9.git"
 	TKG_REPO = params.TKG_REPO ? params.TKG_REPO : "https://github.com/AdoptOpenJDK/TKG.git"
+	//JDK has different repo for different version, setting default as AdoptOpenJDK openjdk-jdk8u or  empty so we could parse it depends on other parameters?
+	//For Zos, user should set up right repo something like: git@github.ibm.com:runtimes/openj9-openjdk-jdk**-zos.git	
```

Comment:
We should replace https to ssh for users if it is zos. We can do it in this file (i.e., line 39) or in build.xml. I would prefer to do it in build.xml as it also covers local runs.

Reply:
My original thought was if JDK_URL is defined by user then it's user's responsibility to provide the correct repo url either it's for zos or non-zos. For user-defined it includes two scenarios:
 1. upstream pass in ( upstream's responsibility to pass in correct repo based on the platform or os)
 2. local or grinder user.

If the JDK_URL is not provided by user then this case is figured out by build.xml
https://github.com/AdoptOpenJDK/openjdk-tests/blob/3930ce93585320167de07cf9f9e1fa3d9b5c8f45/openjdk/build.xml#L128

Reply:
I agree this is user responsibility. But it would be nice if we could cover this case for the users. Just like we did for other repos. i.e., ADOPTOPENJDK_REPO, OPENJ9_REPO, TKG_REPO, and SYSTEM repos.

Reply:
I'd also like a consistent / predictable approach for all repos (and users get stuck on this z/os ssh switch all the time... frustrating for both them and us who has to help them troubleshoot it each time).

Reply:
Updated.